packages/openneuro-app/src/sass/crn_app/_dashboard.datasets.scss

```diff
@@ -356,8 +395,28 @@
                         color: $c-33;
                     }
                 }
+                @media only screen and (max-width: 765px) {
+                    .date {
+                        display: none;
+                    }
+                }
+            }
+            @media only screen and (max-width: 765px) {
+                .metric-container {
+                    width: auto;
+                    float: none;
+                    box-sizing: border-box;
+                    padding: 10px 2px 0px 0px;
+                    align-items: center;
+                }
+            }
+            @media only screen and (max-width: 765px) {
+                .status-container {
+                    display:none;
+                }
             }
```

Comment:
Media-queried .metric-container and .status-container styles could be bundled together since they share the same type, feature, and level.